runtime/src/bank/tests.rs

```diff
@@ -268,9 +268,9 @@ fn test_bank_unix_timestamp_from_genesis() {
         genesis_config.creation_time,
         bank.unix_timestamp_from_genesis()
     );
-    let slots_per_sec = 1.0
-        / (duration_as_s(&genesis_config.poh_config.target_tick_duration)
-            * genesis_config.ticks_per_slot as f32);
+    let slots_per_sec = (genesis_config.poh_config.target_tick_duration.as_secs_f32()
+        * genesis_config.ticks_per_slot as f32)
+        .recip();
```

Comment:
Similar to other, I simplified this to use `recip()`; can revert it if desired.

Reply:
My first thought about the switch to `recip()` on a floating-point was to be concerned about loss of precision, but since both are estimated timings in tests, not critical.